edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/Constants.cs

```diff
@@ -82,5 +82,7 @@ public static class EdgeDeployment
         public const string RunAsNonRootKey = "RunAsNonRoot";
 
         public const string UnknownImage = "unknown";
+
+        public const string HostIPC = "host";
     }
```

Comment:
Do we set **host** as the default ipc mode?

Reply:
Default depends on the Docker daemon, but the typical default is "private" if it's not set.  I.e. each docker container gets its own shared memory.  K8s has a boolean, so it either "hostIPC" or not. I assumed the false case defaults to "private", but did not confirm.